src/plugins/intel_cpu/thirdparty/CMakeLists.txt

```diff
@@ -34,11 +34,9 @@ function(ie_add_onednn)
     set(DNNL_ENABLE_PRIMITIVE "CONVOLUTION;DECONVOLUTION;CONCAT;LRN;INNER_PRODUCT;MATMUL;POOLING;REDUCTION;REORDER;RNN;SOFTMAX" CACHE STRING "" FORCE)
     set(DNNL_ENABLE_WORKLOAD "INFERENCE" CACHE STRING "" FORCE)
 
-    if (ENABLE_CPU_DEBUG_CAPS)
-        set(DNNL_VERBOSE "ON" CACHE STRING "" FORCE)
-    else()
-        set(DNNL_VERBOSE "OFF" CACHE STRING "" FORCE)
-    endif()
+    # Allow to enable oneDNN verbose mode by default since no user / model sensitive information is printed
```

Comment:
Not sure we need this comment. Would prefer to remove at all.

Reply:
Then the comment itself is not valid. We have this option enabled because it is enabled by default inside oneDNN while user information sensitivity is quite a controversial topic

Reply:
If we rely on oneDNN default behavior then we should not explicitly configure this option at all.